Sprint-1/interpret/percentage-change.js

```diff
@@ -13,10 +13,20 @@ console.log(`The percentage change is ${percentageChange}`);
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
 
+/* 5 function calls in lines: 4,5,10 */
+
 // b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
 
+/* SyntaxError in line 5 */
+
 // c) Identify all the lines that are variable reassignment statements
 
+/* 4,5 */
+
 // d) Identify all the lines that are variable declarations
 
+/* 1,2 */
```

Comment:
Any other lines with variable declarations? 🤔

Reply:
Yes, thanks, forgot 7 and 8 )